packages/common/src/types/savedCharts.ts

```diff
@@ -75,17 +95,7 @@ export type Series = {
     areaStyle?: {};
     showSymbol?: boolean;
     smooth?: boolean;
-    markLine?: {
-        data: Record<string, string>[];
-        lineStyle: {
-            color: string;
-            width: number;
-            type: string;
-        };
-        label: {
-            formatter?: string;
-        };
-    };
+    markLine?: MarkLine;
```

Comment:
What is the backwards compatibility of this change?

Reply:
good question. it should be backwards compatible with the existing reference lines. I structured the types a bit better, and I added color and formatter to each markLineData.

Reply:
I've just tested this. I created a chart with reference line and saved on master. 
![Screenshot from 2023-01-10 16-13-03](https://user-images.githubusercontent.com/1983672/211589065-76a42d00-5ca6-4c9e-89fe-64240e8d2141.png)

Then I switched to my branch, and the chart was loaded with the same reference line. 

![Screenshot from 2023-01-10 16-13-33](https://user-images.githubusercontent.com/1983672/211589160-1a43a10d-4bc1-485d-85e7-1da60b575a56.png)

Reply:
Fixed 

![image](https://user-images.githubusercontent.com/1983672/211593924-1e9cf9ef-5d87-47ad-96ec-1804d36b3b06.png)